### DISCUSSION TITLE
稀土掘金助手 v1.7.1 已发布

### DISCUSSION THREAD

**Discussion by iDerekLi** [iDerekLi/juejin-helper#105] [Announcements]
- 🥳 添加模拟用户活跃功能
- 🐲 工作流性能优化

👾 前往 [juejin-helper](https://github.com/iDerekLi/juejin-helper) Fetch你的仓库代码哦!

**Comment by Jason11Q** [comment]
现在还能用吗？好像一直登录失败

**Reply by iDerekLi** [reply]
可以用，如果登陆失败，尝试重新登陆掘金，在获取请求中的Cookie试试

**Reply by Jason11Q** [reply]
昨天更新一次Cookie，今天更新一次，这两天都是失败啊

**Reply by iDerekLi** [reply]
你是怎样获得Cookie的

**Reply by Jason11Q** [reply]
我用的count那个接口，之前用的这个没问题啊，不过之前是部署到阿里云了，现在是用的github，我再看看，是不是我COOKIE写的位置不对，我是加到Action那个环境了

**Reply by iDerekLi** [reply]
你COOKIE变量填写在github上有问题，估计是多了某些符号，比如说换行符、回车符。

**Reply by Jason11Q** [reply]
搞定了，之前那个秘钥和变量没有分那么多类好像，现在分了三类，那个Actions我以为是给工作流专用的，就加到那里了，换到工作区环境就可以了。

**Reply by iDerekLi** [reply]
嗯嗯，好用了就行
